pkg/infrastructure/gcp/clusterapi/clusterapi.go

```diff
@@ -226,9 +226,18 @@ func (p Provider) DestroyBootstrap(dir string) error {
 	if err != nil {
 		return err
 	}
-	if err := gcp.DestroyStorage(context.Background(), metadata.ClusterID); err != nil {
+	if metadata.GCP == nil {
+		return fmt.Errorf("DestroyBootstrap called for non-GCP cluster: %w", err)
+	}
+	ctx := context.Background()
```

Comment:
DestroyBootstrap now has a context parameter:

https://github.com/openshift/installer/blob/ae21a7628080f601541ec80c3b08083cb670b98e/pkg/infrastructure/clusterapi/clusterapi.go#L360

So you should be able to rebase and just delete this line.

Reply:
Fixed.